include/openssl/ssl.h

```diff
@@ -775,6 +775,25 @@ __owur int SSL_extension_supported(unsigned int ext_type);
 # define SSL_MAC_FLAG_READ_MAC_STREAM 1
 # define SSL_MAC_FLAG_WRITE_MAC_STREAM 2
 
+/*
+ * A callback for logging out TLS key material. This callback should log out
+ * |line| followed by a newline.
+ */
+typedef void(*ssl_ctx_keylog_callback)(const SSL *ssl, const char *line);
```

Comment:
I was saying : "Add a space between void and the parenthesis" ... like you do fix the pod file :
Syntax is ok, but it is more readable with it.